python/ml4ir/base/features/preprocessing.py

```diff
@@ -290,6 +291,34 @@ def convert_label_to_clicks(label_vector, dtype):
     clicks = tf.dtypes.cast(cond, typ)
     return clicks
 
+@tf.function
+def convert_fr_to_one_hot(fr_value, max_ranks, mask_fr):
+    """Convert the label vector to binary clicks. Documents with the maximum labels are considered clicked and receive
```

Comment:
An example on this docstring would help. 
Also, your `max_ranks` variable explanation is very good. Why not call it `max_docs_per_query` directly?